src/PrestaShopBundle/Controller/Admin/Sell/Catalog/MonitoringController.php

```diff
@@ -247,4 +261,51 @@ private function getProductWithoutPriceGrid(ProductWithoutPriceFilters $filters)
 
         return $gridFactory->getGrid($filters);
     }
+
+    /**
+     * Toggle product status
+     *
+     * @AdminSecurity(
+     *     "is_granted(['update'], request.get('_legacy_controller'))",
+     *     message="You do not have permission to update this."
+     * )
+     *
+     * @param $productId
+     *
+     * @return JsonResponse
+     */
+    public function toggleStatusAction($productId)
```

Comment:
Shall we put this action in MonitoringController ... or in ProductController 🤔 ?

I think ProductController is maybe better ? For example is in the products listing we have a toggle action too, we will want to reuse this action and it would be strange that a Products Listing calls an ajax route in MonitoringController.